examples/src/test/scala/io/finch/todo/TodoSpec.scala

```diff
@@ -0,0 +1,83 @@
+package io.finch.todo
+
+import java.util.UUID
+
+import com.twitter.finagle.http.Status
+import com.twitter.io.Buf
+import io.circe.generic.auto._
+import io.circe.syntax._
+import io.finch.Input
+import org.scalatest.{FlatSpec, Matchers}
+
+class TodoSpec extends FlatSpec with Matchers {
+  import Main._
+
+  behavior of "the postTodo endpoint"
+  case class TodoWithoutId(title: String, completed: Boolean, order: Int)
+  it should "create a todo" in {
+    val todoWithoutId = TodoWithoutId("foo", completed = false, 0)
+    val input = Input.post("/todos")
+      .withBody(Buf.Utf8(todoWithoutId.asJson.toString), Some("application/json;charset=utf8"))
```

Comment:
I'd probably vote for using scalacheck here and generate random `Todo`s. What do you think?